File: CB14_Alarms/app/src/main/java/com/technicalrj/cb14_alarms/MainActivity.java
package com.technicalrj.cb14_alarms;

import android.app.AlarmManager;
import android.app.PendingIntent;
import android.content.Intent;
import android.os.SystemClock;
import android.support.v4.content.ContextCompat;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        AlarmManager alarmManager = (AlarmManager) getSystemService(ALARM_SERVICE);

        Intent i = new Intent(this,Main2Activity.class);
        PendingIntent pi = PendingIntent.getActivity(this,123,i,PendingIntent.FLAG_UPDATE_CURRENT);
        //alarmManager.set(AlarmManager.ELAPSED_REALTIME, SystemClock.elapsedRealtime() +10000,pi);

        alarmManager.setRepeating(AlarmManager.ELAPSED_REALTIME_WAKEUP,SystemClock.elapsedRealtime()+10000,60000,pi);




    }
}

File: CB10_File_SQL/app/src/main/java/com/technicalrj/cb10_file_sql/MainActivity.kt
package com.technicalrj.cb10_file_sql

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v4.content.ContextCompat
import kotlinx.android.synthetic.main.activity_main.*
import java.io.File
import java.io.FileOutputStream

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)




        button.setOnClickListener{

            var dataDir = ContextCompat.getDataDir(this@MainActivity)
            var file = File(dataDir,"sample.txt")
            file.appendText(editText.text.toString())
        }

        button2.setOnClickListener{

            var dataDir = ContextCompat.getDataDir(this@MainActivity)
            var file = File(dataDir,"sample.txt")
            textView.text = file.readText()
        }

    }
}
